src/Slider/README.md

```diff
@@ -0,0 +1,86 @@
+# Slider
+To implement Slider component into your project you'll need to add the import:
+```jsx
+import Slider from "@kiwicom/orbit-components/lib/Slider";
+```
+
+After adding import into your project you can use it simply like:
+```jsx
+<Slider defaultValue={5} onChange={(value) => doSomething(value)} />
+```
+
+## Props
+Table below contains all types of the props available in the Slider component.
+
+| Name                  | Type                                  | Default         | Description                     |
+| :-------------------- | :------------------------------------ | :-------------- | :------------------------------ |
+| ariaLabel             | `Translation or Translation[]`        |                 | `aria-label` attribute or attributes for handles. See [functional specs](#functional-specs).
+| ariaValueText         | `Translation`                         |                 | Readable text alternative of current value (aria-valueNow).
```

Comment:
This in ideal case won't be used at all, it's just for edge cases and it should be changed when value is changed. This would probably deserves better documentation.